lib/spack/spack/test/cmd/module.py

```diff
@@ -174,6 +172,7 @@ def test_loads_recursive_excluded(database, module_configuration, config_name):
 writer_cls = spack.modules.lmod.LmodModulefileWriter
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Symlinks cannot overwrite files on windows")
```

Comment:
Will this still be true after the symlink changes land?

Reply:
I can't remember exactly why I placed this comment. Looking at it now, the most obvious blocker for the test is true symlinks not being available for non-admin users which the tests will be run on. I updated the comment to reflect that.